lockbox-ios/Presenter/ItemListPresenter.swift

```diff
@@ -129,6 +136,21 @@ class ItemListPresenter: BaseItemListPresenter {
         self.setupSpinnerDisplay()
 
         let itemSortObservable = self.userDefaultStore.itemListSort
+        
+        itemSortObservable.bind(to: self.listSortedObserver).disposed(by: self.disposeBag)
```

Comment:
I'm not an Rx expect, but is it ok to re-use the same `asObservable` object here? 

```swift
self.userDefaultStore.itemListSort
```

gives you a new `Observable` each time.

Reply:
I am pretty sure it's OK to bind/drive the same observable more than once, but hopefully @sashei has a better idea of whether that is OK.

Reply:
Yes, we do that in several other places in the app.